src/containers/mobile-connect-wallets-drawer/store/slice.ts

```diff
@@ -0,0 +1,26 @@
+import { createSlice } from '@reduxjs/toolkit'
```

Comment:
We can use our modals slice & hooks for this rather than creating an entirely new slice :)